src/style.css

```diff
@@ -1 +1,75 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use `*` selector for styles reseting, only for `box-sizing` to apply it to all elements